src/components/flat-table/flat-table-row-header/flat-table-row-header.component.tsx

```diff
@@ -74,6 +75,10 @@ export const FlatTableRowHeader = ({
     [isExpandableCell, onClick]
   );
 
+  const handleOnFocus = useCallback((ev: React.FocusEvent<HTMLElement>) => {
```

Comment:
Same question as above

Reply:
This has been refactored and the unnecessary `useCallback` has been removed in order to increase performance.